pinot-common/src/main/java/org/apache/pinot/common/exception/QueryException.java

```diff
@@ -63,19 +63,27 @@ public static void setMaxLinesOfStackTrace(int maxLinesOfStackTrace) {
   public static final int COMBINE_GROUP_BY_EXCEPTION_ERROR_CODE = 600;
   public static final int QUERY_VALIDATION_ERROR_CODE = 700;
   public static final int UNKNOWN_ERROR_CODE = 1000;
+  public static final int SEGMENTS_NOT_ONLINE_ERROR_CODE = 1001;
+  public static final int SEGMENTS_NOT_ACQUIRED_ERROR_CODE = 1002;
```

Comment:
```suggestion
  public static final int SERVER_SEGMENT_MISSING_ERROR_CODE = 235;
```

Reply:
Hey @Jackie-Jiang may are these error codes suggested by you random or is there some logic to this?

Reply:
They are ordered in query execution phases. E.g currently error code 200-250 is server error; 300+ is broker error